### ISSUE TITLE
Audio not playing on any videos in MacOS 14.5 with KhanAcademy ZIM on Kiwix 3.3.0, the weblink doesn't work either in Safari but works in Firefox?

### ISSUE THREAD

**Issue by coopfidgets** [kiwix/kiwix-apple#804]
I'm not sure what the issue is (it could be the ZIM file) but I am using Kiwix 3.3.0 on MacOS 14.5 and I downloaded the khan academy zim file here (khanacademy_en_all_2023-03.zim)
https://download.kiwix.org/zim/other/https://download.kiwix.org/zim/other/

I also tried to play a random video using the library.kiwix.org link:
https://library.kiwix.org/viewer#khanacademy_en_all_2023-03/d824e26284a65d46b7fe1cf8330122b9

I cannot get the audio to work - Is this Kiwix issue or is it a ZIM file issue?

If I run the link using Firefox then it works fine - is this something to do with the built-in browser for Kiwix? Why does the library.kiwix.org link work fine in Firefox but not in Safari (17.5)?

Is the built-in browser for Kiwix webkit based? Can this be changed?

**Comment by benoit74** [comment]
Nope, kolibri scraper is broken for now, we need to fix the multithreading issue and associated problems (which are most likely just a consequence of multithreading problem, tbc), cf. https://github.com/openzim/kolibri/milestone/7

**Comment by coopfidgets** [comment]
Ok thanks, will wait for an update - will this problem happen again after issue is fixed?

**Comment by kelson42** [comment]
> Ok thanks, will wait for an update - will this problem happen again after issue is fixed? 

No.

**Comment by kelson42** [comment]
It does not seem so, this is actually blocked by https://github.com/openzim/kolibri/issues/101

**Comment by coopfidgets** [comment]
hi there, the latest version still has the same issue, the audio doesn't play on videos in the kiwix app, (I notice it's the same problem as before it doesn't work using Safari but does work using Firefox)
what is causing this?

**Comment by kelson42** [comment]
@coopfidgets What are the exact ZIM the file you have difficulties to read?

**Comment by benoit74** [comment]
Khan academy files have not been updated since March 2023.

And scraper is not yet fixed, issue mentioned above is still there, plus we probably have to update video.JS and fix a bunch of other things - vp9 instead of vp8, ... - we now know are impacting kiwix-apple reader regarding audio/video.

From my PoV, nothing new and nothing left to be done at reader level, we are just struggling to find resources / time to work on the scraper.

**Comment by kelson42** [comment]
@benoit74 Do we have an upstream issue we could consider as duplicate of this? I would like to close this issue as there is nothing we can do more here.

**Comment by kelson42** [comment]
Maybe https://github.com/openzim/kolibri/issues/127
